server/lib/notifications.js

```diff
@@ -117,6 +120,19 @@ async function notifyUserId(UserId, activity, options = {}) {
     const parentCollective = await event.getParentCollective();
     const ics = await event.getICS();
     options.attachments = [{ filename: `${event.slug}.ics`, content: ics }];
+    const transaction = await models.Transaction.findOne({
+      where: { OrderId: activity.data.order.id, type: 'DEBIT' },
```

Comment:
Not sure if it has any impact, but in other places we take the credit transaction

```suggestion
      where: { OrderId: activity.data.order.id, type: 'CREDIT' },
```

Reply:
I wondered that. The DEBIT amount looks weird but in my mind it made sense that the DEBIT was coming out of the user's account, vs the CREDIT being applied to the collective... will fix!